test/__snapshots__/index.test.js.snap

```diff
@@ -2745,3 +2745,279 @@ exports[`fixtures build ts-module with microbundle 7`] = `
+              worker_b.js
+            public-path
+              a.js
+              build
+                runner.html
+              config.json
+              entry.js
+            simple-bundle
+              a.js
+              build
+                runner.html
+              entry.js
+            single-default
+              a.js
+              build
+                runner.html
+              entry.js
+            url-import-meta-worker
+              a.js
+              b.js
+              build
+                runner.html
+              entry.js
+              worker.js
+            worker
+              a.js
+              b.js
+              build
+                runner.html
+              entry.js
+              worker.js
+          import-meta-worker.test.js
+          import-meta.test.js
+          import-worker-url-custom-scheme.test.js
+          import-worker-url.test.js
+          module-worker.test.js
+          more-workers.test.js
+          public-path.test.js
+          simple-bundle.test.js
+          single-default.test.js
+          url-import-meta-worker.test.js
+          worker.test.js
+    ejs
+      LICENSE
+      README.md
+      ejs.js
+      ejs.min.js
+      jakefile.js
+      lib
+        ejs.js
+        utils.js
+      package.json
+      postinstall.js
+    json5
+      CHANGELOG.md
+      LICENSE.md
+      README.md
+      dist
+        index.js
+        index.min.js
+        index.min.mjs
+        index.mjs
+      lib
+        cli.js
+        index.d.ts
+        index.js
+        parse.d.ts
+        parse.js
+        register.js
+        require.js
+        stringify.d.ts
+        stringify.js
+        unicode.d.ts
+        unicode.js
+        util.d.ts
+        util.js
+      package.json
+    magic-string
+      CHANGELOG.md
+      LICENSE
+      README.md
+      dist
+        magic-string.cjs.js
+        magic-string.cjs.js.map
+        magic-string.es.js
+        magic-string.es.js.map
+        magic-string.umd.js
+        magic-string.umd.js.map
+      index.d.ts
+      package.json
+    minimist
+      LICENSE
+      example
+        parse.js
+      index.js
+      package.json
+      readme.markdown
+      test
+        all_bool.js
+        bool.js
+        dash.js
+        default_bool.js
+        dotted.js
+        kv_short.js
+        long.js
+        num.js
+        parse.js
+        parse_modified.js
+        proto.js
+        short.js
+        stop_early.js
+        unknown.js
+        whitespace.js
+    sourcemap-codec
+      CHANGELOG.md
+      LICENSE
+      README.md
+      dist
+        sourcemap-codec.es.js
+        sourcemap-codec.es.js.map
+        sourcemap-codec.umd.js
+        sourcemap-codec.umd.js.map
+        types
+          sourcemap-codec.d.ts
+      package.json
+    tippex
+      CHANGELOG.md
+      README.md
+      dist
+        tippex.es.js
+        tippex.es.js.map
+        tippex.umd.js
+        tippex.umd.js.map
+      package.json
+      src
+        index.js
```

Comment:
not sure why node_modules is being included here - maybe forgot to delete it before running the fixture?

Reply:
oh, that could be. I'll take a look

Reply:
Should be resolved in f155eee